maec_package_schema.xsd

```diff
@@ -923,40 +1009,33 @@
 		</xs:annotation>
 		<xs:attribute name="tool_idref" type="xs:QName"/>
 	</xs:complexType>
-	<xs:complexType name="MalwareNameType">
+	<xs:complexType name="MalwareAliasType">
 		<xs:annotation>
-			<xs:documentation>The MalwareNameType captures name information associated with a malware instance.</xs:documentation>
+			<xs:documentation>The MalwareAliasType captures name information associated with an alias.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element minOccurs="0" name="Malware_Instance_Name"
-				type="maecPackage:NameConfidenceType">
+			<xs:element minOccurs="0" name="Name" type="maecPackage:NameType">
 				<xs:annotation>
 					<xs:documentation>The Malware_Instance_Name field captures the name of the malware instance that is characterized by the Malware Subject.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
-			<xs:element minOccurs="0" name="Malware_Family_Name"
-				type="maecPackage:NameConfidenceType">
-				<xs:annotation>
-					<xs:documentation>The Malware_Family_Name field captures the name of the malware family to which the malware instance characterized by the Malware Subject belongs.</xs:documentation>
-				</xs:annotation>
-			</xs:element>
 			<xs:element minOccurs="0" name="Source" type="maecPackage:SourceType">
 				<xs:annotation>
 					<xs:documentation>The Source field specifies the source of the malware instance and family names captured, respectively, in the Malware_Instance_Name and Malware_Family_Name fields.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 		</xs:sequence>
 	</xs:complexType>
-	<xs:complexType name="NameConfidenceType">
+	<xs:complexType name="NameType">
 		<xs:annotation>
-			<xs:documentation>The NameConfidenceType captures the relative confidence of a malware instance or family name.</xs:documentation>
+			<xs:documentation>The NameType captures the name of a malware instance and the relative confidence of the name.</xs:documentation>

```

Comment:
This should be updated to state that it can capture names of either malware families or malware instances.